docs/products/kafka/howto/create-topics-automatically.md

```diff
@@ -2,74 +2,86 @@
-    production environments as it provides better control over topic
-    settings such as partition count, replication factor, and retention
-    time.
-2.  Enable topic automatic creation: While simpler, enabling automatic
-    topic creation carries some drawbacks. It introduces the risk of
-    inadvertently creating new topics, especially in the case of typos.
-    Additionally, it may result in topics created with
-    [default configuration values](set-kafka-parameters) defined at the service level.
+:::note
+When tiered storage is activated for your Aiven for Apache Kafka service, all new topics
+will have tiered storage enabled by default.
+[Learn more about tiered storage](/docs/products/kafka/concepts/kafka-tiered-storage).
+:::
+
+## Enable automatic topic creation
 
-## Enable automatic topic creation using Aiven Console
+<Tabs groupId="setup">
+<TabItem value="Console" label="Console" default>
 
 To enable automatic topic creation through the Aiven Console, follow
 these steps:
 
-1.  In the [Aiven Console](https://console.aiven.io/), select your
-    project and then choose your Aiven for Apache Kafka® service.
-2.  In the service page, select **Service settings** from the sidebar.
-3.  On the **Service settings** page, scroll down to the **Advanced
-    configuration** section, and click **Configure**.
-4.  In the **Advanced configuration** dialog, click **Add configuration
-    options**.
-5.  Find the `auto_create_topics_enable` parameter and set it to true to
-    enable automatic topic creation.
-6.  Select **Save configuration**.
+1. Log in to the [Aiven Console](https://console.aiven.io/), select your
+   project, and choose your Aiven for Apache Kafka® service.
+1. Click <ConsoleLabel name="service settings"/> from the sidebar.
+1. Scroll down to the **Advanced configuration** section, and click **Configure**.
+1. In the **Advanced configuration** dialog, click
+   <ConsoleLabel name="addadvancedconfiguration"/>.
+1. Find the `auto_create_topics_enable` parameter and set it to true to
+   enable automatic topic creation.
+1. Click **Save configuration**.
 
 :::warning
 Even when you enable automatic topic creation, the user account that
 produces a message to a non-existing topic must have `admin`
 permissions. Aiven for Apache Kafka validates the access control list
 (ACL) before creating the topic. To change user permissions, go to
-the **Users** tab in your service detail page in the [Aiven web
-console](https://console.aiven.io/).
+the **Users** tab in your service detail page in the [Aiven Console](https://console.aiven.io/).
 :::
 
-## Enable automatic topic creation with Aiven CLI
+</TabItem>
+<TabItem value="CLI" label="CLI">
+
+To enable the automatic creation of topics on an existing Aiven for Apache Kafka service,
+use the [Aiven CLI service update command](/docs/tools/cli/service-cli#avn-cli-service-update):
+
+1. Run the following command, replacing `SERVICE_NAME` with the name of your service:
```

Comment:
How about removing the numbered list here since there is just one step here?

```suggestion
To enable the automatic creation of topics on an existing Aiven for Apache Kafka service,
run the [Aiven CLI service update command](/docs/tools/cli/service-cli#avn-cli-service-update), replacing `SERVICE_NAME` with the name of your service:
```